qiita_pet/templates/change_lost_pass.html

```diff
@@ -8,7 +8,7 @@
 {%end%}
 
 {% block content %}
-  {% if not message %}
+  {% if display_body %}
```

Comment:
Not really, the idea of checking the message was to know if an error
occurred and do not display the rest of the page, please look at the
code. The current maintenance mode do not allow for code changes or
restarting the DB so we always need to stop and start the service,
thus maintenance mode is not actually doing what you think it does. If
you want to chat, just let me know.

What about the test?